src/main/java/net/minecraftforge/common/config/ConfigManager.java

```diff
@@ -117,7 +118,24 @@ public static void loadData(ASMDataTable data)
         }
     }
 
-    public static void load(String modid, Config.Type type)
```

Comment:
This breaks bin compat, if you're renaming the method, toss a bouncer in with a //TODO remove comment